cdk.tf/README.md

```diff
@@ -0,0 +1,3 @@
+# cdk.tf
+
+This domain currently only is used for short urls linking to various docs and places and are used in e.g. error messages or across social media.
```

Comment:
Can you expand this please? I'd like to see

- How does it go from here to production?
- What can go wrong? Where do I see if a rollout failed?

Reply:
added a bit more content below. Does that suffice?

Reply:
Ok, yeah works for me